dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/domain/LogSerializerTest.kt

```diff
@@ -318,19 +320,22 @@ internal class LogSerializerTest {
             if (info.id.isNullOrEmpty()) {
                 doesNotHaveField(LogAttributes.USR_ID)
             } else {
-                hasField(LogAttributes.USR_ID, info.id)
+                hasField(LogAttributes.USR_ID, info.id!!)
```

Comment:
Why did you add the `!!`? It doesn't add anything, and the test will fail with a NPE instead of AssertException.

Reply:
It's actually ktlint or kotlint compiler complaining about that. Basically the new NetworkInfo class is public and this test class is Internal and it thinks that this variable can be changed from outside the package. I will see if I can get rid of this by using a local variable here or just ignore ktlint if it's him that complains about.